pkg/definition/gen_sdk/_scaffold/go/go.sum

```diff
@@ -0,0 +1,2292 @@
+4d63.com/gochecknoglobals v0.0.0-20201008074935-acfc0b28355a/go.mod h1:wfdC5ZjKSPr7CybKEcgJhUOgeAQW1+7WcyK8OvUilfo=
```

Comment:
how to run `go mod tidy` for it? I think it's probably get larger when used for long.

Reply:
As long as the `go.mod` is in sub-directory of project. The go embed won't include this directory. That's why I renamed it to `go.mod_`.

We can add some CI check to this sub module by temporarily renaming it back to `go.mod`

Reply:
sure, please do that. You can add a test to generate the sdk and test it, BTW do the go mod tidy to check diff.

Reply:
added
- go mod tidy 
- check diff